dockertest/output/texttable.py

```diff
@@ -18,7 +18,7 @@ class ColumnRanges(Mapping):
     """
 
     # Too few pub. methods, pylint doesn't count abstract __special_methods__
-    # pylint: disable=R0903
+    # pylint: disable=R0903, W0231
```

Comment:
This may be easier to fix in one-line than to ignore with two comment-lines (see below)